src/goose/specs.clj

```diff
@@ -197,12 +203,19 @@
                      :args (s/* ::args-serializable?))
         :ret map?)
 
+(s/fdef c/perform-batch
+        :args (s/cat :opts ::client-opts
+                     :batch-opts ::batch-opts
+                     :execute-fn-sym ::fn-sym
+                     :args (s/and ::batch-args))
```

Comment:
question: why do we need `s/and` here?

Reply:
We don't need `and` here. Earlier, we were composing the `(s/* ::args-serializable?)` condition with `::batch-args` which is now moved into `::batch-args` itself.
It has been left behind by mistake, removed it.